conf/base.config

```diff
@@ -79,6 +77,7 @@ process {
 
 params {
   // Defaults only, expecting to be overwritten
+  singleEnd = true
```

Comment:
Default parameter values should be specified in `nextflow.config`. You can remove this line here.

Reply:
Also, would be nice to replace the `withName` selectors for `withLabel` to simplify this file:
https://github.com/nf-core/chipseq/blob/master/conf/base.config

Reply:
ok, I create process_low and process_medium